themes/helvetica/post.scss

```diff
@@ -59,6 +59,7 @@
       padding: 0;
       margin: 0 4px 4px 21px;
       float: left;
+      max-width: 590px;
```

Comment:
![image](https://cloud.githubusercontent.com/assets/198656/7733488/ef8638ea-fef4-11e4-9148-0ae29bcd5bc2.png)
Hmm, it's not working for me in latest Chrome under mac os x